Eradication/Migration.cpp

```diff
@@ -265,13 +267,6 @@ namespace Kernel
 
     void MigrationInfoFixedRate::NormalizeRates( std::vector<float>& r_rate_cdf, float& r_total_rate )
     {
-        //  Calculate total migration rate
```

Comment:
took this out so I can use it in SaveRawRates without having to calculate it inside of there, but I can undo this.